pypeit/reduce.py

```diff
@@ -354,9 +356,17 @@ def run(self, basename=None, ra=None, dec=None, obstime=None,
                               show=self.reduce_show & (not self.std_redux),
                               manual_extract_dict=manual_extract_dict)
 
+        # Check if the user wants to overwrite the skymask with a pre-defined sky regions file
+        skymask_init, usersky = self.load_skyregions(skymask_init)
+
+        # Define the trim regions for different pypelines
```

Comment:
This is not allowed, since it violates the inheritance structure that we set up for this class. extract is general, and is supposed to handle all cases. If you need to change that and do something IFU specifici, make a child that overloads it. I think what you want to achieve here with edge trimming can be trivially accomodated via a parset which would be the better approach.

Reply:
I agree. I've handled this with a parset now.